core/nwb.image.yaml

```diff
@@ -87,18 +88,22 @@ groups:
       - num_files
       shape:
       - null
-      doc: Each entry is the frame number (within the full ImageSeries) of the first
-        frame in the corresponding external_file entry. This serves as an index to
-        what frames each file contains, allowing random access.Zero-based indexing
-        is used. (The first element will always be zero).
+      doc: Each external image may contain one or more consecutive frames of the full
+        ImageSeries. This attribute serves as an index to indicate which frames each file
+        contains, to faciliate random access. The 'starting_frame' attribute, hence,
+        contains a list of frame numbers within the full ImageSeries of the first frame
+        of each file listed in the parent 'external_file' dataset. Zero-based indexing is
+        used (hence, the first element will always be zero). For example, if the
+        'external_file' dataset has three paths to files and the first file has 5 frames,
+        the second file has 10 frames, and the third file has 20 frames, then this
+        attribute will have values [0, 5, 15].
   - name: format
     dtype: text
-    doc: Format of image. If this is 'external' then the field external_file contains
-      the path or URL information to that file. For tiff, png, jpg, etc, the binary
-      representation of the image is stored in data. If the format is raw then the
-      fields bit_per_pixel and dimension are used. For raw images, only a single channel
-      is stored (e.g., red).
-    quantity: '?'
+    doc: Format of image. If this is 'external', then the attribute 'external_file'
+      contains the path information to the image files. If this is 'raw', then the raw
+      (single-channel) binary data is stored in the 'data' dataset. If this attribute
+      is not present, then then the default format='raw' case is assumed.
```

Comment:
Should we add `default_value: 'raw'`?

Reply:
Adding a `default_value: 'raw'`  is fine, but we should not make the field required in order to not break backward compatibility.